README.md

```diff
@@ -488,7 +488,7 @@ x-logging:
 services:
   northstar1:
     << : *logging
-    image: ghcr.io/pg9182/northstar-dedicated:1-tf2.0.11.0-ns1.4.0
+    image: ghcr.io/pg9182/northstar-dedicated:1-tf2.0.11.0
```

Comment:
I was pinning the version since the config might need to be updated for future NS versions and I want to limit the amount of changes needed for each update.

Reply:
yeah I also thought about setting the version, but I was kinda lost finding the version number for it ^^